glossary.md

```diff
@@ -55,6 +55,11 @@ controls. The more PCV the better for the protocol and its users.
 Protocol Owned Liquidity, is the amount of LP the treasury owns and
 controls. The more POL the better for the protocol and its users.
 
+### PoR
+
+Proof of Reserve, is the mechanism of acumulating protocol controlled value.
+OlympusDAO is designed for PoR by acumulating POL.
```

Comment:
```suggestion
Proof of Reserve, is the mechanism of accumulating PCV.
OlympusDAO is designed for PoR by accumulating POL.
```

Reply:
I think I am out of my depth here. But i thought PoR means some kind of attestation that OlympusDAO has enough reserve to back all the OHM in circulation.

Reply:
I think this PoR is the service Olympus can offer to other protocols

Reply:
```suggestion
Proof of Reserve, is the mechanism of strengthening the reserve of OlympusDAO treasury via the sales of bonds. Bonders provide liquidity to the treasury, thereby building its reserve. In return for their service, bonders get paid in OHM.
```

Reply:
Re-reading Zeus' tweet on PoR, this is my current understanding on the topic